changelog/issue-6343.md

```diff
@@ -0,0 +1,5 @@
+audience: worker-deployers
+level: patch
+reference: issue 6343
+---
+Livelog: improves I/O performance by reading up to 32KiB (previously 4KiB).
```

Comment:
Note, it would be great if we could actually get some before/after benchmarks, to make sure our claims are valid.